workflow/exp/exp.conus12km

```diff
@@ -41,11 +38,12 @@ export LBC_INTERVAL=1
 export LBC_UNGRIB_GROUP_TOTAL_NUM=1
 export LBC_GROUP_TOTAL_NUM=1
 
-export RUN_PERIOD="2024052700-2024052800"
-export CYCLEDEF_IC="   &STARTYEAR;&STARTMONTH;&STARTDAY;0000 &ENDYEAR;&ENDMONTH;&ENDDAY;2300 06:00:00 "
-export CYCLEDEF_LBC="  &STARTYEAR;&STARTMONTH;&STARTDAY;0000 &ENDYEAR;&ENDMONTH;&ENDDAY;2300 06:00:00 "
-export CYCLEDEF_PROD=" &STARTYEAR;&STARTMONTH;&STARTDAY;0000 &ENDYEAR;&ENDMONTH;&ENDDAY;2300 01:00:00 "
-export CYCL_HRS_COLDSTART="00 12"
+export RETRO_PERIOD="2024052700-2024052800"
+export CYCLEDEF_IC_STEP_HRS=6   # do ic jobs every step hours
+export CYCLEDEF_LBC_STEP_HRS=6  # do lbc jobs every step hours
```

Comment:
Do you also need "CYCLEDEF_PROD_STEP_HRS"?

Reply:
NO, we already have a variable for that, i.e `export CYC_INTERVAL=1`